src-java/testing/functional-tests/src/main/groovy/org/openkilda/functionaltests/helpers/SwitchHelper.groovy

```diff
@@ -752,4 +753,71 @@ class SwitchHelper {
     static SwitchPropertiesDto getCachedSwProps(SwitchId switchId) {
         return getCachedAllSwProps().find { it.switchId == switchId }
     }
+
```

Comment:
This part has been added from https://github.com/telstra/open-kilda/pull/5481/files and will not be a part of this PR after merging PR 5481 to develop. And this part was verified in the scope of HA-Flow server42 PR(can be skipped during review).